app/views/api/v2/auth_source/main.json.rabl

```diff
@@ -0,0 +1,5 @@
+object @auth_source
+
+extends "api/v2/auth_source/base"
+
+attributes :onthefly_register, :usergroup_sync, :tls, :server_type, :created_at, :updated_at
```

Comment:
I have found records of this attributes in database due to I think it make senes to show that. Which is related for all authsorces ? - :created_at, :updated_at ? Some else ?